src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -38,9 +39,7 @@
 
 HOME = os.path.expanduser("~")
 GLOBAl_CONFIG_LOCATION = os.path.join(HOME, ".zowe")
```

Comment:
This seems like a typo from a previous PR:

```suggestion
GLOBAL_CONFIG_LOCATION = os.path.join(HOME, ".zowe")
```

I'm guessing that this will have to be renamed in multiple places.